connectivity/FEATURE_BLE/libraries/ble-api-implementation/source/GapImpl.cpp

```diff
@@ -2066,8 +2142,12 @@ ble_error_t Gap::setPeriodicAdvertisingParameters(
         advertiseTxPower
     );
 }
+#endif
+#endif
 
 
+#if BLE_ROLE_BROADCASTER
+#if BLE_FEATURE_PERIODIC_ADVERTISING
```

Comment:
would all these make sense to be `#if ... && .... ` rather than these if if ?

Reply:
my changes mirror how it is done in the header, if we change it here to `#if ... && ....` it should also be changed in the header

Reply:
I see, all fine then from my side

Reply:
Out of curiosity, what would you improve in the headers and cpp files ?